src/codegeneration/PlaceholderParser.js

```diff
@@ -20,7 +20,6 @@ import {
 } from '../syntax/trees/ParseTreeType.js';
 import {IdentifierToken} from '../syntax/IdentifierToken.js';
 import {LiteralToken} from '../syntax/LiteralToken.js';
-import {Map} from '../runtime/polyfills/Map.js';
```

Comment:
I was torn about this one. We already depend on a global Map in other places.

The difference due to this change is that we get the builtin if provided; if not we get it implicitly through the polyfill.